Game/Waypoint.h

```diff
@@ -1,12 +1,12 @@
 
 
-#ifndef WayPoint_H
-#define WayPoint_H
+#ifndef Waypoint_H
+#define Waypoint_H

```

Comment:
Do we want to switch these over to #pragma once over time?

Reply:
Yes. Was thinking of doing later as a lower priority task, though maybe I should just go do it.